openstack/bms/v2/servers/list.go

```diff
@@ -33,7 +33,7 @@ var (
 	SortAvailabilityZone SortKey = "availability_zone"
```

Comment:
This is used anywhere?

Reply:
This is the way to implement enumeration in golang. (A very bad thing in golang.)
Whether it is used or not, it is best to write it out as long as it is defined in the API.